Makefile

```
@@ -1888,8 +1888,14 @@ help:
 	@echo  '  modules         - default target, build the module(s)'
 	@echo  '  modules_install - install the module'
 	@echo  '  clean           - remove generated files in module directory only'
+	@echo  '  rust-analyzer	  - Generate rust-project.json rust-analyzer support file'
```

I appreciate that you copied the line as-is from the other side, but let's be consistent with the casing of the other lines here (I agree these should also just follow the rest, but that would be a different patch, for the kbuild tree probably).

```suggestion
	@echo  '  rust-analyzer	  - generate rust-project.json rust-analyzer support file'
```